handler/config_set.go

```diff
@@ -0,0 +1,10 @@
+package handler
+
+// ConfigSet contains overrides for individual hosts
+type ConfigSet struct {
+	Name string
```

Comment:
It's good practice to label all fields to avoid struct name changes breaking existing configs